src/components/patientPage/patientCard/index.tsx

```diff
@@ -16,42 +16,27 @@ const PatientCard = ({ patient }: Props) => {
   getPatientNumberCard and getSubjectNumberCard are now rendering PatientInfo elements with click option which print the results on the console.
   */
   const getPatientNumberCard = (
-    object: "observations" | "conditions",
+    object:
+      | "observations"
+      | "conditions"
+      | "allergyIntolerances"
+      | "episodesOfCare",
     writtenName: string
   ) => {
+    let resourceNumber: string = "";
     if (patient[object]) {
-      return (
-        <div
-          onClick={() => {
-            console.log(writtenName + " : ", patient[object].entry);
-          }}
-        >
-          <PatientGeneralInfo
-            type={writtenName}
-            content={patient[object].total.toString()}
-          />
-        </div>
-      );
-    }
-  };
+      resourceNumber = patient[object].length;
+    } else resourceNumber = "0";
```

Comment:
Also, in this case you can use a ternary operator which is easier to read:
`const resourceNumber = patient[object] ? patient[object].length : 0`

Reply:
done